README.md

```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
 
 ## Recent Releases
 
-#### SDL Protocol Spec: 5.4.1
```

Comment:
Recent Releases should be in chronological order, so Protocol Spec 5.4.1 and RPC Spec 8.0.0 should be listed before SDL Core 8.1.0.

Reply:
done.